website/src/pages/download.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+
+import { Column, Container, Row } from '../components/Grid';
+import Sponsors from '../components/Sponsors';
+
+
+const LATEST_VERSION = '5.0.0-alpha15';
+
+
+const DownloadLink = ({ arch }) => {
+	let filename, label;
+	switch (arch) {
+	case 'macos':
+		filename = 'macosx.tar.gz';
+		label = 'MacOS';
+		break;
+	case 'linux':
+		filename = 'linux.tar.gz';
+		label = 'Linux';
+		break;
+	case 'src':
+		filename = 'src.zip';
+		label = 'Source Code';
+		break;
+	case 'windows':
+		filename = 'windows.zip';
+		label = 'Windows';
+		break;
+	}
+	return (
+		<div className="download-link">
+			<Link to={`https://github.com/premake/premake-core/releases/download/v${LATEST_VERSION}/premake-${LATEST_VERSION}-${filename}`}>
+				<b>{label}</b>
+			</Link>
+		</div>
```

Comment:
I don't think that using `div` here and styling it in this way is a good idea. I suggest removing `div` so the  `Link` becomes the top element. To understand my point of view please look at my comments below.